src/main/kotlin/com/beust/kobalt/maven/PomGenerator.kt

```diff
@@ -55,7 +52,7 @@ public class PomGenerator @Inject constructor(@Assisted val project: Project) {
 
         val buildDir = KFiles.makeDir(project.directory, project.buildDirectory!!)
         val outputDir = KFiles.makeDir(buildDir.path, "libs")
-        val mavenId = MavenId.create(project.group!!, project.artifactId!!, project.packaging, project.version!!)
+        val mavenId = MavenId(project.group!!, project.artifactId!!, project.version!!, project.packaging)
```

Comment:
But it's not a requirement. Since all these arguments are strings, I think there is too much risk to accidentally pass them in the wrong order.